data/api/client-server/login.yaml

```diff
@@ -109,10 +109,13 @@ paths:
               properties:
                 type:
                   type: string
-                  enum:
-                    - m.login.password
-                    - m.login.token
-                  description: The login type being used.
+                  description: |-
+                    The login type being used.
+
+                    This must be a type returned in one of the flows of the
+                    response of the [`GET /login`](/client-server-api/#get_matrixclientv3login)
+                    endpoint, like `m.login.password`, `m.login.token` or
+                    `m.login.sso`.
```

Comment:
I might be confusing something but I think `m.login.sso` is never to be found in `POST /login`, clients should redirect to a dedicated endpoint instead. Am I smoking stuff?

Reply:
Yes, I think that's right.

Reply:
Indeed, I got influenced by the issue without thinking. It is fixed now.